src/OpenTelemetry.Extensions.Hosting/OpenTelemetryBuilder.cs

```diff
@@ -123,126 +109,7 @@ public OpenTelemetryBuilder WithTracing()
-    /// cref="ILoggerProvider"/> named 'OpenTelemetry' into the <see
-    /// cref="IServiceCollection"/>.</item>
-    /// </list>
-    /// </remarks>
-    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
-    /// calls.</returns>
-    internal
-#endif
-        OpenTelemetryBuilder WithLogging()
-        => this.WithLogging(configureBuilder: null, configureOptions: null);
-
-#if EXPOSE_EXPERIMENTAL_FEATURES
-    /// <summary>
-    /// Adds logging services into the builder.
-    /// </summary>
-    /// <remarks><inheritdoc cref="WithLogging()" path="/remarks"/></remarks>
-    /// <param name="configure"><see cref="LoggerProviderBuilder"/>
-    /// configuration callback.</param>
-    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
-    /// calls.</returns>
-#if NET8_0_OR_GREATER
-    [Experimental(DiagnosticDefinitions.LoggerProviderExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
-#endif
-    public
-#else
-    /// <summary>
-    /// Adds logging services into the builder.
-    /// </summary>
-    /// <remarks><inheritdoc cref="WithLogging()" path="/remarks"/></remarks>
-    /// <param name="configure"><see cref="LoggerProviderBuilder"/>
-    /// configuration callback.</param>
-    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
-    /// calls.</returns>
-    internal
-#endif
-        OpenTelemetryBuilder WithLogging(Action<LoggerProviderBuilder> configure)
-    {
-        Guard.ThrowIfNull(configure);
-
-        return this.WithLogging(configureBuilder: configure, configureOptions: null);
-    }
-
-#if EXPOSE_EXPERIMENTAL_FEATURES
-    /// <summary>
-    /// Adds logging services into the builder.
-    /// </summary>
-    /// <remarks><inheritdoc cref="WithLogging()" path="/remarks"/></remarks>
-    /// <param name="configureBuilder">Optional <see
-    /// cref="LoggerProviderBuilder"/> configuration callback.</param>
-    /// <param name="configureOptions">Optional <see
-    /// cref="OpenTelemetryLoggerOptions"/> configuration callback.</param>
-    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
-    /// calls.</returns>
-#if NET8_0_OR_GREATER
-    [Experimental(DiagnosticDefinitions.LoggerProviderExperimentalApi, UrlFormat = DiagnosticDefinitions.ExperimentalApiUrlFormat)]
-#endif
-    public
-#else
-    /// <summary>
-    /// Adds logging services into the builder.
-    /// </summary>
-    /// <remarks><inheritdoc cref="WithLogging()" path="/remarks"/></remarks>
-    /// <param name="configureBuilder">Optional <see
-    /// cref="LoggerProviderBuilder"/> configuration callback.</param>
-    /// <param name="configureOptions">Optional <see
-    /// cref="OpenTelemetryLoggerOptions"/> configuration callback.</param>
-    /// <returns>The supplied <see cref="OpenTelemetryBuilder"/> for chaining
-    /// calls.</returns>
-    internal
-#endif
-        OpenTelemetryBuilder WithLogging(
```

Comment:
What's the overall sequence for landing this work?

Could removing `WithLogging` methods in this PR be premature?

Without them there's no way to really use the new `WithLogging` methods that have moved to the SDK until additional work is landed, correct?

Reply:
Had an offline conversation with @alanwest about this.

If you had before...

```csharp
services.AddOpenTelemetry().WithLogging();
```

...you now need to do...

```csharp
using OpenTelemetry;
services.AddOpenTelemetry().WithLogging();
```

For that reason we decided to put back the `WithLogging` methods on `OpenTelemetryBuilder` for now. We might do more work on follow-ups though.